lib/smart_answer_flows/marriage-abroad/outcomes/payment_information_partials/_pay_by_cash_or_debit_or_credit_card.govspeak.erb

```diff
@@ -0,0 +1 @@
+You can pay at your appointment using a credit or debit card or cash in the local currency.
```

Comment:
Although the wording is slightly different, I think this might already covered by `pay_by_cash_in_local_currency_or_credit_card_not_cheque` - perhaps worth checking with the content designer?

Reply:
I created a new one because it seemed different enough to me. Both mention cash and credit card, but one mentions debit card and no cheque, the other mentions cheque and no debit card.
I'll double check with the content designer.

Reply:
The decision was to keep the newly created payment information partial.